runtime/sema/string_type.go

```diff
@@ -305,3 +317,23 @@ var StringTypeFromCharactersFunctionType = NewSimpleFunctionType(
 	},
 	StringTypeAnnotation,
 )
+
+var StringTypeSplitFunctionType = &FunctionType{
+	Parameters: []Parameter{
+		{
```

Comment:
It might be nice to just have this function be an instance function on `String` values, like in Swift, Kotlin, JavaScript, etc.

Reply:
Done